master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/AbstractMetaManager.java

```diff
@@ -138,6 +138,12 @@ public void updateAppHeartbeatMeta(String appId, long time, long totalWritten, l
     partitionTotalFileCount.add(fileCount);
   }
 
+  public Set<String> getActiveAppIds() {
```

Comment:
Better to use appHeartbeatTime map to get active appIds. Master may encounter corner case.

Reply:
IMO, We can directly use appHeartbeatTime keySet.

Reply:
> IMO, We can directly use appHeartbeatTime keySet.

+1, one application may have many shuffleKeys